cylc/flow/task_pool.py

```diff
@@ -1268,6 +1268,8 @@ def spawn_on_output(self, itask, output, forced=False):
             forced: If True this is a manual spawn command.
 
         """
+        if not itask.flow_nums:
+            return
```

Comment:
The method does more than just spawning downstream tasks so putting this at the top breaks things.